### PR TITLE
Added Swift for TensorFlow

### PR DESCRIPTION
**Pull request by miwojc**
Swift for TensorFlow is a next generation platform for deep learning and differentiable programming.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Awesome Artificial Intelligence, Machine Learning and Deep Learning as we learn
     - [Programming in Python](Programming-in-Python.md)
     - [More...](julia-python-and-r.md#more)
   - [JavaScript](README-details.md#javascript)
+  - [Swift for TensorFlow](https://www.tensorflow.org/swift/) - next generation platform for deep learning and differentiable programming
```
Comment (neomatrix369) [inline README.md]: You mean Tensorflow for Swift? Can you please fix it!

**Comment by neomatrix369** [comment]
Thanks @miwojc for the contribution, just a minor correction to the text

File: README.md (hunk 1 above)
Comment (neomatrix369) [reply README.md]: I see the main link also says Swift for Tensorflow - although previous releases of Tensorflow were titled like on this page https://github.com/tensorflow/tensorflow/tree/master/tensorflow/java

Comment (miwojc) [reply README.md]: Yeah, they call it `Swift for TensorFlow` as it's not a TensorFlow API for Swift, it's a bigger project to build a next generation platform for deep learning and differentiable programming.

**Comment by neomatrix369** [comment]
Let's merge it and get along with things

@miwojc thanks again, keep them coming!

**Pull request merged**